src/core/Highlighter.ts

```diff
@@ -196,8 +218,17 @@ export default class Highlighter {
    *
    * Returns the list of newly-drawn highlights.
    */
-  redraw = (annotation) => {
-    this.undraw(annotation);
-    return this.draw(annotation);
+  redraw = (id: number|string, range: Range.SerializedRange, annotationData: any) => {
+    this.undraw(id);
+    return this.draw(id, range, annotationData);
+  }
+
+  destroy = () => {
```

Comment:
this function is probably possible to rewrite in pure js.

If I understand it correctly sth like the fragment below should be fine
```
this.element.querySelectorAll(`.${this.options.highlightClass}`).forEach( (el) => {
  const parentNode = el.parentNode; 
  const childNodes = el.childNodes;
  childNodes.forEach(node => parentNode.insertBefore(node, el));
  el.remove();
});
```